src/main/kotlin/com/improve_future/harmonica/core/table/TableBuilder.kt

```diff
@@ -782,7 +828,7 @@ class TableBuilder {
     fun refer(
         tableName: String,
         nullable: Boolean = true,
-        default: Long? = null,
+        default: Int? = null,
```

Comment:
@Sylux6 
Thanks again, but this part has to use `Long`, because SQLite uses 64 bits for integer.

Reference
https://www.sqlite.org/datatype3.html

Reply:
Reverted back